.github/workflows/update_contributor_map.yml

```diff
@@ -0,0 +1,19 @@
+name: Update Contributor Map
```

Comment:
Note that this will only work in `main`. I generated the initial map in a different repo.